docs/user/access-control/creating-sample-user.md

```diff
@@ -40,10 +40,10 @@ subjects:
 
 ## Getting a Bearer Token
 
-Now we need to find the token we can use to log in. Execute the following command:
+Now we need to find the token we can use to log in. Execute the following command to create a token with a custom expiration:
 
 ```shell
-kubectl -n kubernetes-dashboard create token admin-user
+kubectl -n kubernetes-dashboard create token admin-user --duration 8760h
```

Comment:
Setting it to one year is not good security practice.

Reply:
Do I need to keep this parameter `--duration` so that the people know that an expiration time can be specified, and then set the time parameter to 24 hours?

Reply:
I don't think we have to mention that. Everyone can look for options using `kubectl create token --help`.

Reply:
ok, i remove it

Reply:
i have already remove `--duration`, and the img `/images/overview.png` is too old, is there needs to use dashboard-ui.png insteads of it?